db/comdb2.c

```diff
@@ -4851,233 +4851,191 @@ void *statthd(void *p)
-                strbuf *logstr = strbuf_new();
-                diff_qtrap = nqtrap - last_report_nqtrap;
-                diff_fstrap = nfstrap - last_report_nfstrap;
-                diff_nsql = nsql - last_report_nsql;
-                diff_nsql_steps = nsql_steps - last_report_nsql_steps;
-                diff_newsql = newsql - last_report_newsql;
-                int diff_newsql_steps = newsql_steps - last_report_newsql_steps;
-                diff_nretries = nretries - last_report_nretries;
-                diff_ncommits = ncommits - last_report_ncommits;
-                diff_ncommit_time = ncommit_time - last_report_ncommit_time;
-
-                diff_bpool_hits = bpool_hits - last_report_bpool_hits;
-                diff_bpool_misses = bpool_misses - last_report_bpool_misses;
-
-                if (diff_qtrap || diff_newsql || diff_nsql || diff_nsql_steps ||
-                    diff_fstrap || diff_bpool_hits || diff_bpool_misses ||
-                    diff_ncommit_time) {
-
-                    strbuf_appendf(logstr, "diff");
-                    if (diff_qtrap || diff_nretries) {
-                        strbuf_appendf(logstr, " n_reqs %d n_retries %d",
-                                       diff_qtrap, diff_nretries);
-                    }
-                    if (diff_nsql) {
-                        strbuf_appendf(logstr, " nsql %d", diff_nsql);
-                    }
-                    if (diff_newsql) {
-                        strbuf_appendf(logstr, " newsql %d", diff_newsql);
-                    }
-                    if (diff_nsql_steps) {
-                        strbuf_appendf(logstr, " nsqlsteps %d",
-                                       diff_nsql_steps);
-                    }
-                    if (diff_newsql_steps) {
-                        strbuf_appendf(logstr, " newsqlsteps %d",
-                                       diff_newsql_steps);
-                    }
-                    if (diff_fstrap) {
-                        strbuf_appendf(logstr, " n_fsreqs %d", diff_fstrap);
-                    }
-                    if (diff_ncommit_time && diff_ncommits) {
-                        strbuf_appendf(logstr, " n_commit_time %lld ms",
-                                       diff_ncommit_time /
-                                           (1000 * diff_ncommits));
-                    }
-                    if (diff_bpool_hits) {
-                        strbuf_appendf(logstr, " n_cache_hits %llu",
-                                       (long long unsigned int)diff_bpool_hits);
-                    }
-                    if (diff_bpool_misses) {
-                        strbuf_appendf(
-                            logstr, " n_cache_misses %llu",
-                            (long long unsigned int)diff_bpool_misses);
-                    }
-                    strbuf_appendf(logstr, "\n");
-                    reqlog_logl(statlogger, REQL_INFO, strbuf_buf(logstr));
+        thresh = reqlog_diffstat_thresh();
```

Comment:
Diff is mostly indentation and changes applied by the formatting script, actual change is just removing the condition. We go from

```c
if (!get_schema_change_in_progress(__func__, __LINE__)) {
  // update statreqs
}
```

to

```c
// update statreqs
```